src/components/plots/ContinuousEmbeddingPlot.jsx

```diff
@@ -82,6 +87,7 @@ const ContinuousEmbeddingPlot = (props) => {
 
     if (!loading
       && !error
+      && config
```

Comment:
Instead, show a loader so we don't have to delay showing the component until `config` is fetched from the internet.